ts/screens/profile/SecurityScreen.tsx

```diff
@@ -101,53 +109,52 @@ const SecurityScreen = (): React.ReactElement => {
       ),
     [dispatch]
   );
-  useEffect(() => {
-    getBiometricsType().then(
-      biometricsType => {
-        setIsFingerprintAvailable(isBiometricsValidType(biometricsType));
-      },
-      _ => undefined
-    );
-  }, []);
 
   // FIXME: Add alert if user refused IOS biometric permission on
   // the first time the app is opened: https://pagopa.atlassian.net/browse/IA-67
```

Comment:
What about this? We are now able to provide this alert? What do you think?

Reply:
I checked the Jira task and it brought me to this [design](https://www.figma.com/design/zzRuoFxorMJmdkOBZH8WRG/Consenso-negato-biometrico?node-id=172-0&t=zO73g7dBWMCN50lu-0). I think we are able to do it now, but I thought it was no necessary since the toggle seems able to activate/deactivate biometric permission. Am I loosing some informations about the reason why this alert should be displayed?

Reply:
I think it'd be a complimentary alert that invite the citizen to allow the biometric for the app. The first time you install the app if you tap "not allow" in the system alert showing up in the biometric activation screen during the onboarding flow, you don't allow the app to use the biometric for the app itself. In this situation when you try to enable the biometric in the "security" screen you should warn the citizen that they'd enable the biometric in the system application settings so that to remember the citizen that this functionality was previously denied. Am I correct @thisisjp ?

Reply:
I got it. Maybe that Figma is trying to handle an edge case: if you tap "not now" in the biometric screen, during an app first install when you enable the biometric in the "security" screen and you tap "not allow" in the system alert, you continue to see the switch, but then if you enable/disable it does not allow to use the biometric, because it's disable in the app system settings. Am I right?

Reply:
BTW we could handle this edge case in a second PR.

Reply:
Ok, as you explained it's clear now! Maybe it's better to handle it in a separate PR, as you already suggested!